CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
 ## Unreleased changes
+
+## 2.0.0
 - Added `GetBlockItemStatus` endpoint to clients and added helper 
```

Comment:
This is the correct way of doing things, but perhaps we could also consider that the other comment subsumes this one in the sense that it states everything is overwritten, and leave it out.

Reply:
For this release we should just have the rewrite, I agree.